chain-impl-mockchain/src/ledger.rs

```diff
@@ -456,7 +457,9 @@ fn input_utxo_verify(
                 ));
             };
 
-            let verified = signature.verify(&xpub, &transaction_id);
+            let data_to_verify =
+                WitnessUTxOData::new(&ledger.static_params.block0_initial_hash, &transaction_id);
```

Comment:
I'd rather rename all `UTxO` to `Utxo`: more in line with Rust naming conventions for acronyms and less keyboard twisting.

Reply:
good spot. Thanks